networkx/generators/classic.py

```diff
@@ -195,6 +196,37 @@ def barbell_graph(m1, m2, create_using=None):
         G.add_edge(m1 + m2 - 1, m1 + m2)
     return G
 
+def binomial_tree(n):
+    """Returns the Binomial Tree of order n.
+
+    Parameters
+    ----------
+
+    n : int
+        Order of the binomial tree.
+
+    Returns
+    -------
+    G : NetworkX graph
+        A binomial tree of $2^n$ vertices and $2^n - 1$ edges.
+
+    Notes
+    -----
+    The binomial tree of order 0 consists of a single vertex.
```

Comment:
Move this info from the Notes section into text one blank line under the first line of the doc_string.
So there will be no Notes section -- only a paragraph or two near the top.